dom/test/browser/src/elements.ts

```diff
@@ -103,9 +98,7 @@ describe('DOMSource.elements()', function() {
     dispose = run();
   });
 
-  it('should return a stream of arrays of elements of size 1 when querying ":root"', function(
-    done,
-  ) {
+  it('should return a stream of arrays of elements of size 1 when querying ":root"', function(done) {
```

Comment:
Was prettier applied on this line or was this formatted manually? It seems to exceed 80 columns

Reply:
Prettier did that